workshop_app/models.py

```diff
@@ -200,26 +200,26 @@ def __str__(self):
         )
 
 
-class ProfileComments(models.Model):
+class WorkshopComment(models.Model):
     """
-    Contains comments posted by instructors on coordinator profile
+    Contains comments posted by users on workshop instances
     """
 
-    coordinator = models.ForeignKey(User,
-                                    on_delete=models.CASCADE)
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
     comment = models.TextField()
-    instructor = models.ForeignKey(User, related_name="%(app_label)s_%(class)s_related", on_delete=models.CASCADE)
+    public = models.BooleanField(default=True)
     created_date = models.DateTimeField(default=timezone.now)
+    workshop = models.ForeignKey(Workshop, on_delete=models.CASCADE)
```

Comment:
I'll need a boolean field called Hide, only instructors should be able to see this field and change this field. If hide is true, only instructors should be able to see that comment, the coordinator will be unable to see. Hidden comments should have a UI component that tells the instructor that it is a hidden comment.

Otherwise all comments (if not hidden) should be visible to all.

Comments made by Coordinators will always be visible to all.

Reply:
Please add a hidden tag to the non public comments in the Template to understand that this is a hidden comment